runestone/fitb/js/fitb.js

```diff
@@ -124,7 +124,13 @@ FITB.prototype.renderFITBFeedbackDiv = function () {
 
 FITB.prototype.restoreAnswers = function (data) {
     // Restore answers from storage retrieval done in RunestoneBase
-    var arr = data.answer.split(",");
+    try {
+        // The new encoding is JSON. Try that first.
+        var arr = JSON.parse(data);
```

Comment:
I get an error on this. But the following seemed to work.
```suggestion
        var arr = JSON.parse(data.answer);
```

Reply:
Oops! Thanks -- that's correct. I'll update my PR.

It looks like @bnmnetp already included part of this. but that code doesn't support old comma-style encoding. Brad, this seems like it would break when trying to restore old answers. Am I missing something?

Reply:
OK, sounds good.